src/ResourceManager/Network/Commands.Network/NetworkWatcher/SetAzureNetworkWatcherConfigFlowLogCommand.cs

```diff
@@ -257,6 +272,34 @@ public class SetAzureNetworkWatcherConfigFlowLogCommand : NetworkWatcherBaseCmdl
         [ValidateNotNull]
         public IOperationalInsightWorkspace Workspace { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Gets or sets the interval (in minutes) which would decide how frequently TA service should do flow analytics.",
+            ParameterSetName = SetFlowlogByResourceWithTAByResource)]
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Gets or sets the interval (in minutes) which would decide how frequently TA service should do flow analytics.",
+            ParameterSetName = SetFlowlogByResourceWithTAByDetails)]
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Gets or sets the interval (in minutes) which would decide how frequently TA service should do flow analytics.",
+            ParameterSetName = SetFlowlogByNameWithTAByResource)]
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Gets or sets the interval (in minutes) which would decide how frequently TA service should do flow analytics.",
+            ParameterSetName = SetFlowlogByNameWithTAByDetails)]
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Gets or sets the interval (in minutes) which would decide how frequently TA service should do flow analytics.",
+            ParameterSetName = SetFlowlogByLocationWithTAByResource)]
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Gets or sets the interval (in minutes) which would decide how frequently TA service should do flow analytics.",
+            ParameterSetName = SetFlowlogByLocationWithTAByDetails)]
+        [ValidateNotNull]
+        [ValidateRange(1, int.MaxValue)]
+        public int? TrafficAnalyticsInterval { get; set; }
```

Comment:
@zwswim nit: `int?` --> `int`

Reply:
Will this affect existing user ? If they donot set the TrafficAnalyticsInterval ?

Reply:
Same comment above.

Reply:
@cormacpayne fix